src/include/daos/drpc_modules.h

```diff
@@ -70,18 +70,19 @@ enum drpc_mgmt_method {
 	DRPC_METHOD_MGMT_PREP_SHUTDOWN		= 218,
 	DRPC_METHOD_MGMT_DEV_STATE_QUERY	= 219,
 	DRPC_METHOD_MGMT_DEV_SET_FAULTY		= 220,
-	DRPC_METHOD_MGMT_LIST_CONTAINERS	= 221,
-	DRPC_METHOD_MGMT_POOL_QUERY		= 222,
-	DRPC_METHOD_MGMT_POOL_SET_PROP		= 223,
-	DRPC_METHOD_MGMT_PING_RANK		= 224,
-	DRPC_METHOD_MGMT_REINTEGRATE		= 225,
-	DRPC_METHOD_MGMT_CONT_SET_OWNER		= 226,
-	DRPC_METHOD_MGMT_EXCLUDE		= 227,
-	DRPC_METHOD_MGMT_EXTEND			= 228,
-	DRPC_METHOD_MGMT_POOL_EVICT		= 229,
-	DRPC_METHOD_MGMT_DRAIN			= 230,
-	DRPC_METHOD_MGMT_GROUP_UPDATE		= 231,
-	DRPC_METHOD_MGMT_DISCONNECT		= 232,
+	DRPC_METHOD_MGMT_DEV_REPLACE		= 221,
```

Comment:
I think it's OK to do this now, but for the record, once we release 1.2 I think these are going to need to be immutable in order to maintain compatibility across releases.

Reply:
ok